http-filter-example/header_processor.cc

```diff
@@ -0,0 +1,58 @@
+#include "header_processor.h"
+
+namespace Envoy {
+namespace Http {
```

Comment:
Nitpick: it looks like Envoy convention is
```
namespace Envoy {
namespace Extensions {
namespace HttpFilters {
namespace YourFilter {
```
https://github.com/envoyproxy/envoy/blob/main/source/extensions/filters/http/buffer/buffer_filter.h#L13C2-L16